config/config.go

```diff
@@ -14,17 +14,19 @@ const (
 )
 
 type Config struct {
-	Host            string
-	Port            int
-	AdminHost       string
-	AdminPort       int
-	LogLevel        string
-	LogDir          string
-	LogFormat       string
-	Accounts        map[string]string
-	EnableAccessLog bool
-	AdminRedis      RedisConf
-	Pool            RedisPool
+	Host                   string
+	Port                   int
+	AdminHost              string
+	AdminPort              int
+	LogLevel               string
+	LogDir                 string
+	LogFormat              string
+	Accounts               map[string]string
+	EnableAccessLog        bool
+	EnableSecondaryStorage bool
+	AdminRedis             RedisConf
+	Pool                   RedisPool
+	SecondaryStorage       SpannerConfig
```

Comment:
This should be a pointer since the secondary storage is an option.

Reply:
adopted